adam-cli uses git-commit-id-plugin which breaks release?

I just downloaded [a release](https://github.com/bigdatagenomics/adam/archive/adam-parent-spark2_2.11-0.20.0.tar.gz) of ADAM and tried to run `mvn package -DskipTests` which failed on the ADAM CLI module with 

```
[ERROR] Failed to execute goal pl.project13.maven:git-commit-id-plugin:2.2.1:revision (default) on project adam-cli-spark2_2.11: .git directory is not found! Please specify a valid [dotGitDirectory] in your pom.xml -> [Help 1]
```

Is this expected?

--- a/pom.xml
+++ b/pom.xml
@@ -195,6 +195,7 @@
             <gitDescribe>
               <always>true</always>
             </gitDescribe>
+            <failOnNoGitDirectory>false</failOnNoGitDirectory>
           </configuration>
         </plugin>
         <plugin>

